shopfloor/services/service.py

```diff
@@ -118,6 +123,9 @@ def _response(self, base_response=None, data=None, next_state=None, message=None
         if message:
             response["message"] = message
 
+        if popup:
```

Comment:
wondering: couldn't this be just a message_type?
Is unlikely that you set a message and a popup.

Reply:
Actually I used a separate concept because it seems we'll have both most of the time.
For instance, everytime we finish a transfer, we show a _message_ "package x transfered". And when we finish a transfer that would release the next transfer, we show the _modal popup_ to say "hey the next transfer Y is ready", but we still want to display "package x transfered" ("underneath" the pop-up maybe, so visible when we click ok, so the same way as when the pop-up is not shown)